src/chrome/content/rules/Caltech.xml

```diff
@@ -150,19 +150,6 @@ Fetch error: http://web.caltech.edu/ => https://web.caltech.edu/: (6, 'Could not
 
 	<securecookie host=".\.caltech\.edu$" name=".+" />
 
-
```

Comment:
Don't forget to remove the related targets :wink:. Right now, they will be caught by the trivial rule at the end of this rule and cause `Invalid certificate` errors.